pkg/controller/csi_handler_test.go

```diff
@@ -696,7 +697,7 @@ func TestCSIHandler(t *testing.T) {
 		{
 			name:           "unknown node -> error",
 			initialObjects: []runtime.Object{pvWithFinalizer()},
-			addedVA:        va(false, fin, ann),
+			addedVA:        va(false, fin, nil),
```

Comment:
Is this change necessary? My tests shows that presence of the annotation does not make any difference, because status is patched, not the object with its finalizers. Having `ann` in the `addedVA` is more correct, as the Patch below expects it's present in the original object.

Reply:
This applies to all `ann` removed from `addedVA` below

Reply:
Yes, agreed. Added ann back.